docs/BREAKING_CHANGES.md

```diff
@@ -0,0 +1,13 @@
+# Breaking Changes
+
+## 0.42.0
+
+* [SQLite] The table column created using `date()` now uses TEXT datatype instead of DATE (which the database mapped internally to NUMERIC type). 
+This applies to the specific `DateColumnType` in all 3 date/time modules and means `LocalDate` comparisons can now be done directly without conversions.
+* [H2, PostgreSQL] Using `replace()` now throws an exception as the REPLACE command is not supported by these databases. 
+  If `replace()` was being used to perform an insert or update operation, all usages should instead be switched to `upsert()`. [See wiki for UPSERT details](https://github.com/JetBrains/Exposed/wiki/DSL#insert-or-update)
```

Comment:
This links to a future anchor as the PR won't be merged until the release is out and the wiki is updated.